### PR TITLE
refactor(components): added the ability to set additional props

### PR DESCRIPTION
**Pull request by gerus-virtru**
### Proposed Changes
_Please use the Jira Key followed by the changes_

- JIRAKEY-1234
  - Some change
  - Another change

### Checklist

- [ ] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation / readme (if appropriate)
- [ ] I have verified that my changes have not introduced new lint errors
- [ ] I have added a GitHub labels to the PR for either _patch_, `minor`, or **major**

### CODE REVIEW

File: lib/components/expirationDate/expirationPicker.js [hunk 1]
```diff
@@ -7,96 +7,103 @@ import styles from './expirationPicker.css';
...
 const { Option } = Select;
 
-const ExpirationPicker = React.memo(({ getTime, isModal, expDateTime }) => {
-  const [number, setNumber] = useState('1');
-  const [type, setType] = useState(0);
-  const [formattedString, setFormatedString] = useState('');
-  const [shouldUpdate, setShouldUpdate] = useState(!isModal || (isModal && !expDateTime));
+const ExpirationPicker = React.memo(
+  ({ getTime, isModal, expDateTime, defaultExpirationRange, numberOfDefaultExpirationTime }) => {
+    const selectValues = [
```
[the remark this answers is not in the document]
Comment (gerus-virtru) [reply lib/components/expirationDate/expirationPicker.js]: fixed

File: lib/components/expirationDate/expirationPicker.js [hunk 2]
```diff
@@ -7,96 +7,103 @@ import styles from './expirationPicker.css';
...
+    );
+  },
+);
 
 ExpirationPicker.propTypes = {
   getTime: PropTypes.func.isRequired,
   isModal: PropTypes.bool,
   expDateTime: PropTypes.string,
+  defaultExpirationRange: PropTypes.string,
+  numberOfDefaultExpirationTime: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (gerus-virtru) [reply lib/components/expirationDate/expirationPicker.js]: fixed

**Review by riabokon-virtru — APPROVED** [review]

**Comment by riabokon-virtru** [comment]
@[gerus-virtru](https://github.com/gerus-virtru) LGTM

**Review by riabokon-virtru — APPROVED** [review]

**Review by jrschumacher — APPROVED** [review]
Approving per @riabokon-virtru

**Pull request merged**